notify-api/src/notify_api/utils/enums.py

```diff
@@ -24,3 +24,14 @@ class Role(BaseEnum):
     SMS = 'sms'
     JOB = 'job'
     INVALID = 'invalid'
+
+
+class MillionverifierResult(BaseEnum):
+    """The result of email verification."""
+
+    OK = 'ok'
```

Comment:
you can use 
```suggestion
    OK = auto()
```
If you look at the sql enum implementation you'll see that it also sets up auto to create a lowercase name of the ENUM.